HousingFinanceInterimApi.Tests/V1/Gateways/ReportGatewayTests.cs

```diff
@@ -552,95 +552,6 @@ await _classUnderTest
         }
         #endregion
 
-        #region Cash Import
```

Comment:
These are redundant now because we're no longer calling this method for this report - there's nothing to mock and the new test runs directly against the Dockerised db